origami.json

```diff
@@ -1,5 +1,5 @@
 {
-	"description": "Buttons module",
+	"description": "Styles for buttons. This component supports all products under the FT, and provides customisation options with the FT colour palette.",
```

Comment:
> ...This component supports all products under the FT...
I don't think we need to say this, feels like fluff to make the description longer and it's not necessarily true if a product decides to not use our buttons